stdpopsim/cli.py

```diff
@@ -418,6 +425,19 @@ def stdpopsim_cli_parser():
             "option are set, the option takes precedence. "
             f"Default: {stdpopsim.get_cache_dir()}"))
 
+    sim_engines = [se.id for se in stdpopsim.all_simulation_engines()]
+    top_parser.add_argument(
+        "-E", "--simulation-engine",
```

Comment:
Maybe just ``--engine``? The ``simulation`` bit could be understood, and this would save us a bit of typing. (Also elsewhere in the API, e.g. ``stdpopsim.all_engines()``. It's an application for running simulations, so it seems OK that an "engine" would be interpreted as a simulation engine.

Happy to discuss other terminology though (you originally preferred 'backend').

Reply:
reading through this PR now. I vote for `--engine`` too

Reply:
Yep, I'll do s/simulation_engine/simulation_engine/ and s/SimulationEngine/Engine/ too for consistency.